### DISCUSSION TITLE
【特殊】叮当草原小铺诚信代肝

### DISCUSSION THREAD

**Discussion by diam5333** [share121/inter-knot#2883] [其他]
![93cb21d31b2e73cca7a60a40cf282b99](https://github.com/user-attachments/assets/75a2420a-76ff-40ee-9868-55f21e5c1b62)
接原神星铁绝区零鸣潮各种游戏代肝，最近快开学没时间的学生有折扣，价格可谈，有意向的可以加v：yanxihan6668，q：2855874842
![3d45e56296e3164ef12168418af9266a](https://github.com/user-attachments/assets/dd7302df-5b04-4969-9837-74665ee762e2)
![462eb05016c29d9960d038fdf7c3d11c](https://github.com/user-attachments/assets/d800e309-fa7b-4992-b3f4-f4751f923d4e)
![f9021e61dd1dc51c5cb618793116256c](https://github.com/user-attachments/assets/1bb059aa-4950-4285-aeb1-aaf6c15fdbe2)
![d40aaa7a7b101f04b8a73dfa7bc7d400](https://github.com/user-attachments/assets/001ea339-01f6-4de5-998b-a909f42bf98f)
![0d9bed1b94425284cd3a7f3b5c76091a](https://github.com/user-attachments/assets/404ddd3e-b224-4de6-8a07-f8250efc8349)

**Comment by FRIKY-c** [comment]
6，这里也有代肝，牛的

**Comment by NiuBoss123** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/10/07/91e7229cfc39dfe2b092c5f458d6b855_8792299537189788152.png" alt="阿库_老人手机" width="75" >

**Comment by diam5333** [comment]
![image](https://github.com/user-attachments/assets/4f48d001-02f2-4b60-8372-6584132193aa)

**Comment by diam5333** [comment]
![image](https://github.com/user-attachments/assets/8197409d-9b70-4ed2-aa83-308f787f04cb)

**Comment by diam5333** [comment]
![Uploading image.png…]()
